server/documents/modules/dropdown.html.eco

```diff
@@ -1681,6 +1681,43 @@ themes      : ['Default', 'GitHub', 'Material']
       </div>
     </div>
 
+    <div class="tag example">
+      <h4 class="ui header">Stuck User Additions</h4>
+      <p>While using <code>allowAdditions: true</code>, the first line indicating "add:" with the user addition may not be correctly displayed, depending on the container size and the distance between topmost visible item and menu's top item.
+
+      <p>You can stick this addition line to the top of the menu by providing the additional <code>stuck</code> class.</p>
+
+      <div class="ui ignored warning message">
+        A stuck user additions dropdown needs a modern browser with <a href="https://caniuse.com/?search=position%20sticky">position:sticky support</a>, so does not work in IE11 or legacy Edge
+      </div>
+
+      <div class="ui sub header">Single</div>
+      <select name="skills" class="ui fluid search dropdown">
+        <%- @partial('examples/single/skill-menu') %>
+      </select>
+
+      <div class="ui sub header">Multiple</div>
+      <div class="ui fluid multiple search selection dropdown">
+        <input name="tags" type="hidden" />
+        <i class="dropdown icon"></i>
+        <div class="default text">Skills</div>
+        <div class="menu">
+          <%- @partial('examples/single/preset-menu') %>
+        </div>
+      </div>
+
+      <div class="evaluated code" data-type="javascript">
+        $('.tag.example .ui.dropdown')
+          .dropdown({
+            allowAdditions: true,
```

Comment:
Add the missing hideAdditions:false setting, otherwise the example wont show the feature
```suggestion
            allowAdditions: true,
            hideAdditions: false,
```